### ISSUE TITLE
Unable to go back and edit response

### ISSUE THREAD

**Issue by rc16** [rw251/code-list-creator#3]
If a user accidentally accepts a code but should have rejected a code, or vice versa, there is no facility to be able to do this.

**Comment by rw251** [comment]
Good point. I will add the ability to go back by pressing a certain key - perhaps "b" for back, "u" for undo, or maybe just <backspace>. Whichever it is I'll make sure it's obvious in the tool.
